[Issue-11449] [docs] Updated Python installation steps for Mac (#11589)
apache/pulsar: site2/docs/client-libraries-python.md

```

### Optional dependencies

> If you are installing the client libraries on Linux then in order to support aspects like pulsar functions or Avro serialization, additional optional components can be installed alongside the  `pulsar-client` library

Reviewer: ```suggestion
If you install the client libraries on Linux to support services like Pulsar functions or Avro serialization, you can install optional components alongside the  `pulsar-client` library.
```
